keyboards/ymdk/oni_hhkb/keyboard.json

```
@@ -0,0 +1,263 @@
+{
+    "keyboard_name": "Oni HHKB",
+    "manufacturer": "YMDK",
+    "url": "",
+    "maintainer": "chrisgve",
+    "usb": {
+        "vid": "0x45D4",
+        "pid": "0x0021",
+        "device_version": "0.0.1",
+        "force_nkro": true
```

Generally shouldn't be force-enabled by default, as NKRO has issues in many BIOS's.
```suggestion
        "device_version": "0.0.1"
```